cypress/integration/smoke.spec.ts

```diff
@@ -50,14 +50,23 @@ e2e.scenario({
 
 e2e.scenario({
   describeName: 'Import dashboard',
-  itName: 'adds JSON',
+  itName: 'fills out datasource connection configuration, imports JSON dashboard, adds panel and run query',
   scenario: () => {
     e2e()
       .readProvisions(['datasources/adx.yaml'])
       .then((ADXProvisions: ADXProvision[]) => {
         addCommonProvisioningADXDatasource(ADXProvisions);
 
         e2e.flows.importDashboard(TEST_DASHBOARD);
+
+        e2e.flows.addPanel({
+          matchScreenshot: false,
+          visitDashboardAtStart: false,
+          queriesForm: () => {
+            e2eSelectors.queryEditor.editKQL.button().click({ timeout: 5000 });
```

Comment:
Somehow I cannot access the monaco editor in e2e tests - getting:

<img width="760" alt="Screenshot 2022-04-04 at 12 16 16" src="https://user-images.githubusercontent.com/42030685/161524225-2ea02104-256c-4a2b-a70f-a05a451ccbd4.png">

Reply:
you hit https://github.com/grafana/azure-data-explorer-datasource/issues/325 :) be sure you don't load any monaco editor prior to this

Reply:
Thanks! Not sure how I can enforce that though, I suppose the other monaco editor is the one used in dashboard upload json, I will try to split the tests.

Otherwise, I am currently working on a workaround that uses the builder instead of the KQL editor...

Reply:
> I suppose the other monaco editor is the one used in dashboard upload json, I will try to split the tests.

I remember that situation triggering the bug so yes. If it's not possible to avoid that, I would disable the JSON upload test while we fix #224, it's less valuable than the one using the editor.

Reply:
I just moved it to a separate test and that works fine, thanks!